src/toxic.c

```diff
@@ -86,7 +86,39 @@
 
 struct Winthread Winthread;
 
-static void queue_init_message(const char *msg, ...);
+__attribute__((format(printf, 2, 3)))
```

Comment:
The attribute is only needed on the declaration (in the header).

Reply:
Done.